### DISCUSSION TITLE
Secret names must not start with GITHUB_. 这个问题您是如何解决的

### DISCUSSION THREAD

**Discussion by candies404** [MartialBE/one-hub#198] [Q&A]
https://github.com/MartialBE/one-api/blob/main/.github/workflows/docker-image.yml 这个文件中74行      password: ${{ secrets.GITHUB_TOKEN }} ，

**Comment by MartialBE** [comment]
你需要在 仓库设置-actions-General-Workflow permissions 配置权限
